lib/test.js

```diff
@@ -56,7 +56,9 @@ const test = (suite, buildConfig = config.defaultBuildConfig, options) => {
     braveArgs.push('--test-launcher-jobs=' + options.test_launcher_jobs)
   }
 
+  const args = util.buildArgsToString(config.buildArgs())
   // Build the tests
+  util.run('gn', ['gen', config.outputDir, '--args="' + args + '"'], config.defaultOptions)
```

Comment:
this is wrong and should be reverted

Reply:
this will wipe out any changes passed on the command line to build and potentially cause unexpected large rebuilds